Tests/AutoLogin/Authentication/Http/FireWall/AutoLoginListenerTest.php

```diff
@@ -0,0 +1,143 @@
+<?php declare(strict_types=1);
+
+namespace Tests\AutoLogin\Http\Firewall;
```

Comment:
Note to self: the prefix here can be changed to `Jmikola\AutoLogin\Tests` before merging and the file path changed to `tests/Http/Firewall/AutoLoginListener.php`.

Additionally, the autoloader configuration in `composer.json` can be updated:

```
    "autoload": {
        "psr-4": { "Jmikola\\AutoLogin\\": "src/" }
    },
    "autoload-dev": {
        "psr-4": { "Jmikola\\AutoLogin\\Tests\\": "tests/" }
    },
```

In doing so, the `src/Jmikola/AutoLogin` directory can also be collapsed to just `src/`.